### ISSUE TITLE
Clean up the panel UI

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#1300]
Now that we know what all the requirements are for the panel UI, clean up the existing UI so that it follows UI patterns that are familiar to users. 
![image](https://user-images.githubusercontent.com/42216982/64575282-d3140280-d327-11e9-8503-a2be5832cc39.png)

**Comment by michelleif** [comment]
this design is great but includes some significant new functionality that I think is beyond the scope of this work cycle (letting users add additional fields and moving template pieces around).

**Comment by michelleif** [comment]
would address #1260

**Comment by astridu** [comment]
@michelleif Yes, it has a lot of snazzy features, but I can prioritize the 'nice-to-haves' lower if we can get the core requirements (around the add/expand/remove) done.

**Comment by michelleif** [comment]
@astridu can you indicate here with an image or a list what the core changes are? and move the nice-to-haves to another ticket(s)?

**Comment by astridu** [comment]
@michelleif I highlighted in green text the core changes, but let me know if this is still too much work. 
![image](https://user-images.githubusercontent.com/42216982/64980531-ddc22080-d86e-11e9-8686-8c7e1d8f59a3.png)

**Comment by michelleif** [comment]
should fix #1236 

for WC2, focus on items in ### green
